powerline/segments/i3wm.py

```diff
@@ -54,12 +54,44 @@ def workspaces(pl, segment_info, only_show=None, output=None, strip=0):
 		'contents': w['name'][min(len(w['name']), strip):],
 		'highlight_groups': calcgrp(w)
 	} for w in conn.get_workspaces()
-		if (not only_show or any(w[typ] for typ in only_show))
-		and (not output or w['output'] == output)
+		if ((not only_show or any(w[typ] for typ in only_show))
+			and (not output or w['output'] == output))
 	]
 
 
 @requires_segment_info
+def workspace(pl, segment_info, workspace=None, strip=0):
```

Comment:
they do not always, but the i3 manual recommends doing it that way. i3 sorts workspaces only if they start with a number like that, but the colon is optional.

>Note that the workspace will hella be named "1: mail". i3 treats workspace names beginning with a number in a slightly special way. Normally, named workspaces are ordered the way they appeared. When they start with a number, i3 will order them numerically. Also, you will be able to use workspace number 1 to switch to the workspace which begins with number 1, regardless of which name it has. This is useful in case you are changing the workspace’s name dynamically. To combine both commands you can use workspace number 1: mail to specify a default name if there’s currently no workspace starting with a "1".

I guess an option that acts like the i3bar options would work though (basically removing `[0-9]+:` if existing, then `str.strip`). What do you mean by strip `.*?`? That would just match everything, no?